dev/com.ibm.ws.messaging.comms.server/src/com/ibm/ws/jfap/inbound/channel/InboundChain.java

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
+ * Copyright (c) 2022, 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package com.ibm.ws.jfap.inbound.channel;
+
+public interface InboundChain {
+	
+	public void enable(boolean enabled);
```

Comment:
don't need the word public here — it's implicit in an interface that all its members are public

Reply:
Did not know this, I will make the change!